### DISCUSSION TITLE
Toolbelt should memorize items for fast switching tools

### DISCUSSION THREAD

**Discussion by io-june** [space-wizards/space-station-14#3971] [Ideaguys]
Colour of hand in toolbelt slot shows that the currently selected item in that hand came from that slot in toolbelt and pressing that key again will stow the item back. Small bar on the bottom of the cell shows type of item that can go there (orange for engineering for example).

**Comment by Acruid** [comment]
Related to https://github.com/space-wizards/space-station-14/issues/18.
